### PR TITLE
reducing size of screenshots for introduction episode

### PR DESCRIPTION
**Pull request by sstevens2**
Related to #51 

Not sure if there are other episodes that need image reduction but I'll look through and make more PRs if so.

### CODE REVIEW

File: _episodes/introduction.md [hunk 1]
```diff
@@ -175,33 +175,34 @@ the website we will be creating.
...
 **or** click the "+" menu in the upper righthand corner and
 choose "New Repository".
 
-![Dropdown plus menu with new repository option highlighted](../fig/plus_new_repo.png)
+<img src="../fig/plus_new_repo.png" alt="Dropdown plus menu with new repository option highlighted" width="200">
 
 Next you will need to fill in some info about your repository.
 
-![Blank new repository page](../fig/blank_new_repo.png)
+<img src="../fig/blank_new_repo.png" alt="Blank new repository page" width=600>
```
Comment (JulianKarlBauer) [inline _episodes/introduction.md:215]: This one looks a bit smaller than the others, although you used consistent width...

**Comment by JulianKarlBauer** [comment]
Looks good to me.

File: _episodes/introduction.md (hunk 1 above)
Comment (sstevens2) [reply _episodes/introduction.md:215]: I'm guessing there is slightly more whitespace around this screenshot

**Comment by unode** [comment]
While going though the intro, I noticed I almost missed a paragraph between screenshots.
There's a continuity of white background so it all blends together...
 
**Can we maybe add an outline to the images, a drop shadow effect or something along these lines?**

screenshot_2020-11-04_01-38-17_467336879

**Comment by unode** [comment]
@sstevens2 The final screenshot includes a green `GitPod` button that is possibly injected by a browser extension.
In my case the "green" button is the ":arrow_down: Code" button, so the difference stood out.

**Comment by sstevens2** [comment]
@unode Are you thinking that the gitpod button should be removed from the screensot?


> @sstevens2 The final screenshot includes a green `GitPod` button that is possibly injected by a browser extension.
> In my case the "green" button is the "⬇️ Code" button, so the difference stood out.

**Comment by sstevens2** [comment]
This will be less of an issue with the resized screenshots as the text is much smaller within the screenshots than in the page.
If it makes it more accessible, we should certainly put boxes around the screenshots though.   Do you know of a way to do this for all of them easily? I don't really know where to start there and wouldn't want to do it manually.

> While going though the intro, I noticed I almost missed a paragraph between screenshots.
> There's a continuity of white background so it all blends together...
> 
> **Can we maybe add an outline to the images, a drop shadow effect or something along these lines?**
> 
> screenshot_2020-11-04_01-38-17_467336879

**Comment by unode** [comment]
> @unode Are you thinking that the gitpod button should be removed from the screensot?

I'd say yes. Perhaps easier to use a browser or profile without extensions for screenshots.
It's a minor thing but given the callout in the introduction requesting that readers open issues if screenshots differ from what they see, this one might already be a candidate :smile:

**Comment by sstevens2** [comment]
I propose we do a screenshot overhaul once the lesson is more stable.  Then we can make them all match (same username) and do it without extensions enabled.

> > @unode Are you thinking that the gitpod button should be removed from the screensot?
> 
> I'd say yes. Perhaps easier to use a browser or profile without extensions for screenshots.
> It's a minor thing but given the callout in the introduction requesting that readers open issues if screenshots differ from what they see, this one might already be a candidate 😄

**Comment by tobyhodges** [comment]
> I propose we do a screenshot overhaul once the lesson is more stable.

@sstevens2 please could you open an issue (with the `wait` label) for this, so we don't forget about it? Then I am happy to merge this.

**Comment by sstevens2** [comment]
@tobyhodges Sure!  Re: discussion below - Do you think we should try to add boundary boxes around the screenshots prior to merging the resizing PRs?

> This will be less of an issue with the resized screenshots as the text is much smaller within the screenshots than in the page.
> If it makes it more accessible, we should certainly put boxes around the screenshots though. Do you know of a way to do this for all of them easily? I don't really know where to start there and wouldn't want to do it manually.
> 
> > While going though the intro, I noticed I almost missed a paragraph between screenshots.
> > There's a continuity of white background so it all blends together...
> > **Can we maybe add an outline to the images, a drop shadow effect or something along these lines?**
> > screenshot_2020-11-04_01-38-17_467336879

**Comment by tobyhodges** [comment]
No. The best way to achieve this would probably be to add a border for every screenshot/figure by adjusting the CSS for the lesson. We're using the remote theme for this lesson, so we would either need to propose the change there, which might be the better option in the long run but regardless would take some time to be merged, or override it locally, which I would consider to be overkill. I suppose there's another option, which is to add an outline to the image files themselves but that feels very hacky.

I will open an issue on the lesson template repository: maybe the maintainers will be in favour of introducing a solution upstream. 

But I am going to merge this PR now.

**Pull request merged**